File: 백준/Bronze/2738. 행렬 덧셈/행렬 덧셈.js
const fs = require('fs');
const input = fs.readFileSync(0, 'utf-8').toString().trim().split('\n').map(x => x.split(' ').map(Number));
const [N, M] = input[0];
const A = input.slice(1, 1 + N);
const B = input.slice(1 + N);
let answer = '';

for (let i = 0; i < N; i++) {
    let row = '';
    
    for (let j = 0; j < M; j++) {
        row += `${A[i][j] + B[i][j]} `;
    }
    
    answer += `${row.trim()}\n`;
}

console.log(answer);

File: 백준/Bronze/2738. 행렬 덧셈/행렬 덧셈.py
N, M = map(int, input().split())
A = [list(map(int, input().split())) for i in range(0, N)]
B = [list(map(int, input().split())) for i in range(0, N)]
C = []

for i in range(0, N):
    row = []
    
    for j in range(0, M):
        row.append(str(A[i][j] + B[i][j]))

    C.append(row)
    
print('\n'.join(map(lambda row: ' '.join(row), C)))